package.json

```diff
@@ -2044,20 +2108,25 @@
         "node-stream-zip": "^1.6.0",
         "onigasm": "^2.2.2",
         "pdfkit": "^0.12.1",
+        "plotly.js": "^1.58.4",
```

Comment:
Is this including plotly twice? This might just be one minified and one-non minified version.

Reply:
yep, and fixed ty